gulpfile.babel.js

```diff
@@ -128,12 +143,71 @@ const buildConfig = (invocationArgs, sourceRoot, testRoot, buildRoot) => {
 
 const config = buildConfig(process.argv, 'source', 'test', 'build');
 
+const httpCache = new Keyv({
+  store: new KeyvFile({
+    filename: `${config.dir.build.cache}/http.json`,
+  }),
+});
+
 // Shared config tasks
 
 const cleanSharedConfig = () => del(config.files.src.derivedConfigs);
 
 export const distributeSharedConfig = gulp.series(cleanSharedConfig, distributeConfig);
 
+// Font tasks
+
+const cleanFonts = () => del(config.files.src.fonts.concat([config.dir.src.sassFonts]));
+
+const compileFonts = () => {
+  fs.ensureDirSync(config.dir.src.fonts);
+
+  const definitions = yaml.safeLoad(fs.readFileSync(config.files.src.fontsDefinition));
+
+  const files = definitions.fonts.reduce(
+    (carry, font) => {
+      font.files.forEach(file => {
+        const uri = url.resolve(definitions.base, file.path);
+        if (!(uri in carry)) {
+          carry[uri] = [];
+        }
+        carry[uri].push({
+          fontDisplay: 'fallback',
+          fontFamily: font.name,
+          fontStyle: file.style || 'normal',
+          fontWeight: file.weight || 400,
+          subsetMap: font.subsets,
+        });
+      });
+
+      return carry;
+    }, {},
+  );
+
+  const fontCache = `${config.dir.build.cache}/fonts`;
+  fs.removeSync(fontCache);
+
+  return Promise.all(Object.keys(files).map(uri =>
+    download(uri, {cache: httpCache})
```

Comment:
Would like to be able to override the TTL, as it's only 300.

Reply:
There's this: https://www.npmjs.com/package/keyv#setkey-value-ttl, not sure whether that's quite what you need though.